gomock.bzl

```diff
@@ -72,9 +66,10 @@ _gomock_source = go_rule(
             doc = "The new Go file to emit the generated mocks into",
             mandatory = True,
         ),
-	"aux_files": attr.string_list_dict(
+        "aux_files": attr.label_keyed_string_dict(
             default = {},
-            doc = "A map from packages to auxilliary Go source files to load for those packages.",
+            doc = "A map auxilliary Go source files to their packages.",
```

Comment:
Accidental?

Reply:
Go source files should be referred to as labels, not strings (see the [implementation](https://github.com/bazelbuild/rules_go/blob/master/go/private/rules/library.bzl#L57) of `go_library` as an example). Labels allow us to refer source files from different packages. However, there is no way to map a string to label in Bazel rules. We can only map a label to a string. I know this will break existing usage, but what other choice do we have?